src/prisma.ts

```diff
@@ -0,0 +1,5 @@
+import { PrismaClient } from '@prisma/client';
```

Comment:
this file is imported instead of re-instantiating the prisma client. (i think there's only supposed to be one instance of the client).